### ISSUE TITLE
Docker image 0.12.11 not compatible systemd using LZ4

### ISSUE THREAD

**Issue by bharrisau** [fluent/fluent-bit#479]
As mentioned in coreos/bugs#2121, newer systemd versions support LZ4. The current fluent bit docker image doesn't support LZ4 and just silently fails to collect any logs.

```
$ docker run -ti --rm -v /var/log/journal:/var/log/journal fluent/fluent-bit:0.12.11 /fluent-bit/bin/fluent-bit -i systemd -p tag='host.*' -p Path=/var/log/journal/ -o stdout  -f 1 -v -v
Fluent-Bit v0.12.11
Copyright (C) Treasure Data

[2018/01/04 08:44:21] [ info] [engine] started
[2018/01/04 08:44:21] [debug] [router] default match rule systemd.0:stdout.0

$ docker run -ti --rm -v /var/log/journal:/var/log/journal fluent/fluent-bit:0.12.11 /fluent-bit/bin/fluent-bit -i systemd -p tag='host.*' -o stdout  -f 1 -v -v
Fluent-Bit v0.12.11
Copyright (C) Treasure Data

[2018/01/04 08:45:37] [ info] [engine] started
[2018/01/04 08:45:37] [debug] [router] default match rule systemd.0:stdout.0

$ docker exec .... journalctl --version
systemd 215
+PAM +AUDIT +SELINUX +IMA +SYSVINIT +LIBCRYPTSETUP +GCRYPT +ACL +XZ -SECCOMP -APPARMOR

$ journalctl --version
systemd 234
+PAM +AUDIT +SELINUX +IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT -GNUTLS -ACL +XZ +LZ4 +SECCOMP +BLKID -ELFUTILS +KMOD -IDN2 -IDN default-hierarchy=legacy
```

When changing the Dockerfile to use `FROM debian:stretch-slim` I now get:

```
docker run -ti --rm -v /var/log/journal:/var/log/journal fluent-test /fluent-bit/bin/fluent-bit -i systemd -p tag='host.*' -p Path=/var/log/journal -o stdout  -f 1
Fluent-Bit v0.12.11
Copyright (C) Treasure Data

[2018/01/04 09:17:22] [ info] [engine] started
[0] host.my.service .........
...
...

$ journalctl --version
systemd 232
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN
```

I'd suggest changing the base image over to `debian:stretch-slim` or something that includes LZ4.

**Comment by cordoval** [comment]
i am not collecting logs on image: fluent/fluent-bit:0.12.9, is that image also affected by this?

**Comment by bharrisau** [comment]
I don't believe any of the current images include LZ4 journal support. You'd need to check your host system to see if it is using LZ4 compression on the journal to see if it is the same issue. You can also try disabling compression in journald.conf to check if that fixes log collection.

**Comment by edsiper** [comment]
thanks for pointing out this issue.

Using the image __debian:stretch-slim__ should be correct. What I am doing now is making the image for Fluent Bit 0.13 based on that one, I will send more instructions shortly.

**Comment by edsiper** [comment]
would you please evaluate the image being used for 0.13-dev ?

https://github.com/fluent/fluent-bit-kubernetes-logging/tree/0.13-dev

**Comment by edsiper** [comment]
ping @cordoval @bharrisau

**Comment by bharrisau** [comment]
Just moved to the `fluent/fluent-bit-0.13-dev:0.4` and host logs are working.

**Comment by edsiper** [comment]
thanks. Closing this ticket as it's already fixed in the upcoming 0.13 docker images
